.github/workflows/release-py.yml

```diff
@@ -0,0 +1,288 @@
+name: release-python
+env:
+  LAYER: py38-layer
+  STACK: py38-release
+  UTILS: ${{ github.workspace }}/opentelemetry-lambda/utils/sam
+
+on:
+  workflow_dispatch:
+    inputs:
+      fromLayer:
+        description: 'Soaked layer name in test account'
+        required: false
+        default: 'py38-layer-86558a5-2fc2419'
+      toLayer:
+        description: 'Layer name publishing to Prod account'
+        required: true
+        default: 'py38-preview'
+      version:
+        description: 'the version number to release'
+        required: true
+        default: '0.1'
+      regions:
+        description: 'regions to be deployed'
+        retuired: true
+        default: "{\"region\": [ \"us-east-1\", \"us-east-2\", \"us-west-1\", \"us-west-2\", \"ap-south-1\", \"ap-northeast-3\", \"ap-northeast-2\", \"ap-southeast-1\", \"ap-southeast-2\", \"ap-northeast-1\", \"ca-central-1\", \"eu-central-1\", \"eu-west-1\", \"eu-west-2\", \"eu-west-3\", \"eu-north-1\", \"sa-east-1\"]}"
+
+
+jobs:
+  cache-layer-from-test:
+    runs-on: ubuntu-latest
+
+    env:
+      KEY: ${{ github.event.inputs.fromLayer }}
+      AWS_REGION: us-west-2
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: adot custom
+        run: |
+          cp -rf adot/* opentelemetry-lambda/
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID_TEST }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY_TEST }}
+          mask-aws-account-id: false
+          aws-region: ${{ env.AWS_REGION }}
+
+      - name: Cache layer
+        uses: actions/cache@v2
+        id: cache-layer
+        with:
+          path: ./layer.zip
+          key: ${{ github.event.inputs.fromLayer }}
+
+      - name: download layer
+        env:
+          OTEL_LAMBDA_LAYER: ${{ github.event.inputs.toLayer }}
+        run: |
+          layerArn=$($UTILS/layer.sh -a)
+          URL=$(aws lambda get-layer-version-by-arn --arn $layerArn --query Content.Location --output text)
+          curl $URL -o layer.zip
+
+  publish-layer:
```

Comment:
Is there any difference in publish-layer between languages?

Reply:
No difference between languages